src/components/Filters/Filters.tsx

```diff
@@ -54,6 +54,10 @@ export interface FilterInterface {
   shortcut?: boolean;
   /** Whether or not the filter is disabled */
   disabled?: boolean;
+  /**
+   * @default true
+   * Whether or not the clear button is displayed */
+  clearButton?: boolean;
```

Comment:
optional parameters that default to true behave somewhat weirdly. This is one of those rare cases where we want a negative in a prop name.

`hideClearButton` or something like that.


Alternatively is there a way we can auto hide this in certain circumstances without needing the extra prop? "don't show a clear if there are no selected filters" or something like that

Reply:
(sorry for the delay) @emmaorhun and I fixed up your first comment, swell suggestion, and we tried a lot on the second bit but it was a bit of a hassle and can get done in a future PR as a separate issue!